frontend/yarn.lock

```diff
@@ -9783,10 +9783,12 @@ hoopy@^0.1.4:
   resolved "https://registry.yarnpkg.com/hoopy/-/hoopy-0.1.4.tgz#609207d661100033a9a9402ad3dea677381c1b1d"
   integrity sha512-HRcs+2mr52W0K+x8RzcLzuPPmVIKMSv97RGHy0Ea9y/mpcaK+xTrjICA04KAHi4GRzxliNqNJEFYWHghy3rSfQ==
 
-hosted-git-info@^2.1.4:
-  version "2.7.1"
-  resolved "https://registry.yarnpkg.com/hosted-git-info/-/hosted-git-info-2.7.1.tgz#97f236977bd6e125408930ff6de3eec6281ec047"
-  integrity sha512-7T/BxH19zbcCTa8XkMlbK5lTo1WtgkFi3GvdWEyNuc4Vex7/9Dqbnpsf4JMydcfj9HCg4zUWFTL3Za6lapg5/w==
+hosted-git-info@^2.1.4, hosted-git-info@^3.0.8:
```

Comment:
I'm a little unsure about the resolutions here since this is a major version bump that could have API incompatibilities.

Reply:
Right. Due to my previous comment I'm wondering if it even makes sense to bump it since we are not directly using the pkg anywhere, which is not making the vulnerable `fromUrl()` method exposed...
Or we need to migrate to `dart-sass`, which seems to me to be a bit of a overkill ...
@spadgett thoughts ?